### PR TITLE
Remove JVM_VERSION calculation as JVM_VERSION is now exported from

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
TestConfig

Signed-off-by: Mesbah <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
@sophia-guo - could you please review the PR?

File: get.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ getBinaryOpenjdk()
 getTestKitGen()
 {
 	cd $TESTDIR
-	git clone https://github.com/eclipse/openj9.git
+	git clone https://github.com/Mesbah-Alam/openj9.git -b addExportForJvmVersion
```
Comment (sophia-guo) [inline get.sh]: This should be updated.

**Comment by sophia-guo** [comment]
Please also update JenkinsfileBase to export JVM_VERSION explicitly.

File: get.sh (hunk 1 above)
Comment (Mesbah-Alam) [reply get.sh]: Yep, done. I had only added it to test it off of my branch in a personal build.

**Pull request merged**